dev/integration_tests/flutter_gallery/web/index.html

```diff
@@ -24,7 +24,7 @@
   https://developers.google.com/web/fundamentals/primers/service-workers -->
   <script>
     if ('serviceWorker' in navigator) {
-      window.addEventListener('load', function () {
+      window.addEventListener('flutter-first-frame', function () {
```

Comment:
this event is dispatched by the flutter binding

Reply:
There's probably some people with the old `addEventListener('load'...)`, how do we normally communicate this type of changes to users? A tweet/blog post/something else?

Reply:
hmmm, its not really a breaking change. Probably the changelog?